qa/standalone/mon/health-mute.sh

```diff
@@ -38,7 +38,7 @@ function TEST_mute() {
     ceph -s
     ceph health | grep HEALTH_OK || return 1
     # test warning on setting pool size=1
-    ceph osd pool set foo size 1
+    ceph osd pool set foo size 1 --yes-i-really-mean-it
```

Comment:
I think you should instead set "mon_allow_pool_size_one = true" somewhere to let these kind of tests pass?

Reply:
I think you should also set "mon_allow_pool_size_one = true" somewhere to let these kind of tests pass?